### PR TITLE
Add enthio javascript.

### PR DESCRIPTION
**Pull request by kylerush**

### CODE REVIEW

**Comment by kylerush** [comment]
@dtothefp can you review?

File: website/index.hbs [hunk 1]
```diff
@@ -5,6 +5,8 @@ body_class:
 page_script: homepage.js
 layout_script: false
 layout_body_class: false
+external_async_script:
+- "//ethn.io/84818.js"
 meta_tags:
```
Comment (dtothefp) [inline website/index.hbs]: @kylerush I think this should just be `async_script` and then in the `wrapper`
```
{{#if page.external_async_script}}
       {{#each page.external_async_script}}
       <script async="true" src="{{#hasAssetPath .}}{{../../assetsDir}}/{{/hasAssetPath}}{{.}}"></script>
       {{/each}}
{{/if}}
```

**Comment by kylerush** [comment]
@dtothefp updated

**Comment by dtothefp** [comment]
@kylerush LGTM 
hipster-hotties-13

**Pull request merged**